dnf-behave-tests/dnf/vars.feature

```diff
@@ -100,3 +100,12 @@ Scenario: Using dnf with non-files in /etc/dnf/vars
       dnf-ci-fedora\s+dnf-ci-fedora test repository
       """
 
+
+@bz2141215
+Scenario: Ignoring variable files with invalid encoding
+  Given I copy file "/usr/lib/sysimage/rpm/rpmdb.sqlite" to "/etc/dnf/vars/releasever"
```

Comment:
The path is a variable and depends on the major version of distribution. Please use other approach how to mess up with encoding.